doc/getting_started/shift_detection.rst

```diff
@@ -0,0 +1,47 @@
+Distribution Shift Detection
+****************************
+
+Retraining or updating of a machine learning model is usually necessitated by *shifts* in the
+distribution of data that is being served to the model.
+Renate provides methods for distribution shift detection that can help you decide when to update
+your model.
+This functionality resides in the :py:mod:`renate.shift` subpackage.
+
+Shift Detector Interface
+========================
+
+The shift detectors in :py:mod:`renate.shift` derive from a common class
+:py:class:`~renate.shift.detector.ShiftDetector`, which defines the main interface. Once a
+:code:`detector` object has been initialized, one calls :code:`detector.fit(dataset_ref)` on a
+reference dataset (a PyTorch dataset object). This reference dataset characterizes the expected
+data distribution. It may, e.g., be the validation set used during the previous fitting of the
```

Comment:
nit: dataset used for training would be easier to understand for inexperienced readers

Reply:
I'm hesitant, because it would be dangerous to use the actual training set if the feature extractor has seen that.